newlib/libc/sys/vita/sys/socket.h

```diff
@@ -169,6 +181,9 @@ struct sockaddr {
 	char		sa_data[14];	/* actually longer; address value */
 };
 
+#define in_addr     SceNetInAddr
+#define sockaddr_in SceNetSockaddrIn
+
```

Comment:
These structures don't belong in sys/socket.h and you cannot define them to a typedef-name structure.